### PR TITLE
Make sure PHPUnit is cased correctly in fixers descriptions

### PR DESCRIPTION
**Pull request by kubawerlos**
With a special [dedication](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3745/files#diff-88b99bb28683bd5b7e3a204826ead112R1126) for @BackEndTea.

### CODE REVIEW

File: tests/AutoReview/FixerTest.php [hunk 1]
```diff
@@ -231,4 +233,13 @@ private static function assertStringIsNotEmpty($actual, $message = '')
         self::assertInternalType('string', $actual, $message);
         self::assertNotEmpty($actual, $message);
     }
+
+    /**
+     * @param mixed  $actual
```
Comment (julienfalque) [inline tests/AutoReview/FixerTest.php]: I would either write `@param string $actual` or add a type check in the method.

Comment (kubawerlos) [reply tests/AutoReview/FixerTest.php]: Fixed, obviously should be `string` (I've copied it from the method above and missed that).

**Comment by julienfalque** [comment]
:+1:

Would you keep PHPDoc or PHPDocs?

**Comment by keradus** [comment]
`PHPDoc`
https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md

File: tests/AutoReview/FixerTest.php [hunk 2]
```diff
@@ -58,7 +58,9 @@ public function testFixerDefinitions(FixerInterface $fixer)
         $fixerIsConfigurable = $fixer instanceof ConfigurationDefinitionFixerInterface;
 
         $this->assertRegExp('/^[A-Z`].*\.$/', $definition->getSummary(), sprintf('[%s] Description must start with capital letter or a ` and end with dot.', $fixerName));
-        $this->assertPhpUnitCasing($definition->getSummary(), sprintf('[%s] `PHPUnit` must be in correct casing in description.', $fixerName));
+        $this->assertNotContains('phpdocs', $definition->getSummary(), sprintf('[%s] `PHPDoc` must not be in the plural in description.', $fixerName), true);
```
Comment (keradus) [inline tests/AutoReview/FixerTest.php]: is it case sensitive or not?
if it is, then it's not enough

File: tests/AutoReview/FixerTest.php [hunk 3]
```diff
@@ -235,11 +239,12 @@ private static function assertStringIsNotEmpty($actual, $message = '')
     }
 
     /**
-     * @param string $actual
+     * @param string $haystack
+     * @param string $needle
      * @param string $message
      */
-    private static function assertPhpUnitCasing($actual, $message)
+    private static function assertCorrectCasing($haystack, $needle, $message)
```
Comment (keradus) [inline tests/AutoReview/FixerTest.php]: expected value, here expected casing, shall be first param, to mimic native PHPUnit's assertions params order

File: tests/AutoReview/FixerTest.php (hunk 2 above)
Comment (kubawerlos) [reply tests/AutoReview/FixerTest.php]: [4th](https://github.com/sebastianbergmann/phpunit/blob/7.1.5/src/Framework/Assert.php#L250) parameter causes it's not.

**Comment by keradus** [comment]
Thank you @kubawerlos.

**Pull request merged**